app/controllers/friendships_controller.rb

```diff
@@ -1,7 +1,17 @@
 class FriendshipsController < ApplicationController
   def create
     friend = User.find_by(email: params[:email])
-    current_user.friendships.create!({ friend_id: friend.id })
-    redirect_to dashboard_path
+    if friend
```

Comment:
I hear ya on the rubocop warning. I'm going to go ahead and merge this to main for now. Since we'll continue to get the rubocop notice we'll be reminded to refactor this somehow.